ecocode-rules-specifications/src/main/rules/EC523/javascript/EC523.asciidoc

```diff
@@ -63,6 +69,11 @@ Location.requestPermissionsAsync(); // Compliant
 
 == Resources
 
+=== Best practises
+
+- https://github.com/cnumr/best-practices-mobile[Mobile-specific Best Practices] by https://collectif.greenit.fr/index_en.html[CNumR]
```

Comment:
Do you have a more precise source?

Reply:
This is the same source as the java version of EC523. Do you have a better source?